angular_token_auth.js

```diff
@@ -13,6 +13,8 @@
         AUTH_HEADER_PREFIX: 'Token',
         ALLOWED_HOSTS: [],
         COOKIE_PATH: null
+        // Optional settings examples:
+        // STORAGE_METHOD: 'localstorage',
```

Comment:
Basically, I see no reason for

```js
// Optional settings examples:
// STORAGE_METHOD: 'localstorage'
```

when you can self-document even further with

```js
// Optional settings examples:
// STORAGE_METHOD: 'localstorage'
// or
// STORAGE_METHOD: 'cookie'
// or
// STORAGE_METHOD: 'noSupport'
```